nupic/regions/SPRegion.py

```diff
@@ -106,15 +103,6 @@ def _buildArgs(f, self=None, kwargs={}):
         argValue = argTuple[2]
       # Set as an instance variable if 'self' was passed in
       setattr(self, argName, argValue)
-      
-  # Translate some parameters for backward compatibility
-  if kwargs.has_key('numActivePerInhArea'):
-    setattr(self, 'numActiveColumnsPerInhArea', kwargs['numActivePerInhArea'])
-    kwargs.pop('numActivePerInhArea')
```

Comment:
maybe write a comment somewhere, eg in `SpatialPooler` that 
"""
# Old params used in obsoleted FDRCSpatialPooler were: 
# deprecated FDRCSpatial -> current SP
# numActivePerInhArea -> numActiveColumnsPerInhArea
# InputBorder -> (unused)
# ...
"""

If someone had a (quite) old script and needed porting to a recent nupic.